docs/source/getting_started.rst

```diff
@@ -132,7 +132,7 @@ Using the connection parameters, you can run this script from a terminal using d
 .. TODO talk more about the parser and how you shouldn't use get_connection()
 
 If your script accepts custom command-line arguments, you can add them to the arguments parser returned by :py:func:`~genestack.make_connection_parser`.
-The arguments ``-u``, ``-p`` and ``-H`` are reserved for the connection parameters.
+The arguments ``-u``, ``-p``, ``-H``, ``--host``, ``--api-token``, ``--show-logs`` and ``--debug`` are reserved for the connection parameters.
```

Comment:
Looks weird to me.  I think the list should look like `--full-option` (`-o`): of course, if there is short option.